pinot-core/src/main/java/org/apache/pinot/core/operator/valsets/TransformBlockValSet.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.pinot.core.operator.docvalsets;
+package org.apache.pinot.core.operator.valsets;
```

Comment:
what's the motivation of this change, seems to me docitset and docvalueset gives me clearer understanding